README.md

```diff
@@ -13,47 +13,156 @@ This is work-in-progress, some things may not work yet.
+   $SSHID -p $PORT root@localhost` without being prompted for a password (where `SSHID` is the SSH
+   identification file and `PORT` is the port that are specified in the `manager` configuration
+   file).
+
+TODO: Describe how to support other types of VM other than QEMU.
+
+### Syzkaller
+
+The syzkaller tools are written in [Go](https://golang.org), so a Go compiler (>= 1.4) is needed
+to build them.  Build with `make`, which generates compiled binaries in the `bin/` folder.
+
+## Configuration
+
+The operation of the syzkaller manager process is governed by a configuration file, passed at
+invocation time with the `-config` option.  This configuration can be based on the
+[example file](manager/example.cfg) `manager/example.cfg`; the file is in JSON format with the
+following keys in its top-level object:
+
+ - `name`: Name to use for this instance.
+ - `http`: URL that will display information about the running manager process.
+ - `master`: Location of the master process that the `manager` should communicate with.
+ - `workdir`: Location of a working directory for the `manager` process. Outputs here include:
+     - `<workdir>/qemu/logN-M-T`: log files
+     - `<workdir>/qemu/imageN`: per-instance copies of the VM disk image
+     - `<workdir>/crashes/crashN-T`: crash output files
+ - `vmlinux`: Location of the `vmlinux` file that corresponds to the kernel being tested.
+ - `type`: Type of virtual machine to use, e.g. `qemu`.
+ - `count`: Number of VMs to run in parallel.
+ - `port`: Port that the manager process listens on for communications from the
+   fuzzer processes running in the VMs.
+ - `params`: A JSON object containing VM configuation, specific to the particular `type` of VM. For
+   `qemu` VMs, this configuration includes:
+      - `kernel`: Location of the `bzImage` file for the kernel to be tested; this is passed as the
+        `-kernel` option to `qemu-system-x86_64`.
+      - `cmdline`: Additional command line options for the booting kernel, for example `root=/dev/sda1`.
+      - `image`: Location of the disk image file for the QEMU instance; a copy of this file is passed as the
+        `-hda` option to `qemu-system-x86_64`.
+      - `sshkey`: Location (on the host machine) of an SSH identity to use for communicating with
+        the virtual machine.
+      - `fuzzer`: Location (on the host machine) of the syzkaller `fuzzer` binary.
+      - `executor`: Location (on the host machine) of the syzkaller `executor` binary.
+      - `port`: TCP port on the host machine that should be redirected to the SSH port (port 22) on
+        the guest VM; this is passed as part of the `hostfwd` option to the `-net` option of
+        `qemu-system-x86_64`.
+      - `cpu`: Number of CPUs to simulate in the VM (*not currently used*).
+      - `mem`: Amount of memory (in MiB) for the VM; this is passed as the `-m` option to
+        `qemu-system-x86_64`.
+ - `disable_syscalls`: List of system calls that should be treated as disabled.
```

Comment:
Please also add 'enable_syscalls' before 'disable_syscalls' for completeness.
enable_syscalls is applied first and allows to select only an interesting subset of syscalls (e.g. open, read, write, close).